packages/e2e-test-utils/src/focusSelectedBlock.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+async function focusSelectedBlock() {
+  // Ideally there shouuld be a UI way to do this. (Focus the selected block)
+  await page.evaluate(() => {
+    wp.data
+      .dispatch('core/block-editor')
+      .selectBlock(
+        wp.data.select('core/block-editor').getSelectedBlockClientId(),
+        0
```

Comment:
What's the `0` here? AFAIK `selectBlock` only takes one argument.

Reply:
Copied from https://github.com/WordPress/gutenberg/blob/8499a38f6d5f0cdfd86c4bf17d42f37232c72125/packages/e2e-test-utils/src/inserter.js#L61-L76

Reply:
Looks like `0` is the default argument here:

https://github.com/WordPress/gutenberg/blob/8499a38f6d5f0cdfd86c4bf17d42f37232c72125/packages/block-editor/src/store/actions.js#L205-L223

Could be safely removed